notebooks/enterprise-search/app-search-engine-exporter.ipynb

```diff
@@ -455,13 +577,15 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Add `sparse_vector` fields for semantic search (optional)\n",
+    "# Add semantic text fields for semantic search (optional)\n",
     "\n",
-    "One of the advantages of having our exported index directly in Elasticsearch is that we can easily take advantage of doing semantic search with ELSER. To do this, we'll need to add a `sparse_vector` field to our index, set up an ingest pipeline, and reindex our data.\n",
+    "One of the advantages of having our exported index directly in Elasticsearch is that we can easily take advantage of doing semantic search with ELSER. To do this, we'll need to add an inference endpoint using ELSER, and a `semantic_text` field to our index to use it.\n",
```

Comment:
Nit: Trying to avoid two uses of `advantage` in the same sentence

```suggestion
    "One of the advantages of exporting our index directly to Elasticsearch is that we can easily perform semantic search with ELSER. To do this, we'll need to add an inference endpoint using ELSER, and a `semantic_text` field to our index to use it.\n",
```